app/controllers/collections_controller.rb

```diff
@@ -65,6 +66,10 @@ def revert
     end
   end
 
+  def download_xml
+    send_data @collection_download, type: 'text/xml', disposition: "attachment; filename=#{@concept_id}.#{@collection_format}"
```

Comment:
I feel like we should be setting the `type` to an appropriate value here, potentially listed in the `DOWNLOAD_XML_OPTIONS` or something. If `.native` is umm-json we'd be telling the file its xml when its really json.

Reply:
this sounds like a good idea. I should have, but overlooked, thinking about if `.native` is umm_json.